apps/console/src/features/organizations/components/add-organization-role-wizard.tsx

```diff
@@ -0,0 +1,470 @@
+/**
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com) All Rights Reserved.
```

Comment:
Need to update the license header to the latest format

Reply:
Fixed in later commits